Git Formats/Git Common.sublime-syntax

```diff
@@ -283,6 +283,7 @@ contexts:
     - include: pretty-formats-wrap-formatting
     - include: pretty-formats-string-formatting
     - include: pretty-formats-trailers
+    - match: \S
```

Comment:
What is that `-match: \S` for? We stay in that context until the `(?!\S)` on line 238 is matched. Removing it doesn't make a difference in your test cases.

Reply:
Good catch! It was added intentionally, along with the Line 394 test. But the test must only have been broken while I was tinkering with it. I'll take that out.